dev/source/docs/gsoc-ideas-list.rst

```diff
@@ -9,17 +9,20 @@ This is a list of projects suggested by ArduPilot developers for `GSoC 2020 <htt
 - :ref:`Non-GPS navigation improvements using Intel RealSense cameras <common-vio-tracking-camera>`
 - :ref:`Object avoidance <common-object-avoidance-landing-page>` improvements for Multicopters and/or Rovers
 - `Optical flow <https://ardupilot.org/copter/docs/common-optical-flow-sensors-landingpage.html>`__ position hold performance improvements for multicopters
-- `Precision Landing <https://ardupilot.org/copter/docs/precision-landing-with-irlock.html>`__ accuracy improvements using `JeVois camera <http://www.jevois.org/>`__ (or similar) for multicopters
-- Automatic docking for cars and boats using `JeVois camera <http://www.jevois.org/>`__ (or similar)
+- `Precision Landing <https://ardupilot.org/copter/docs/precision-landing-with-irlock.html>`__ accuracy improvements for multicopters
+- Automatic docking for cars and boats using camera
+- State estimation from camera or optical flow
 - Lane following or visual follow-me for Copter, Rover or Boat
+- Rover Autotune
 - Walking robot support
 - 3D aerobatic support for fixed wing aircraft
 - Improve :ref:`Morse simulator <sitl-with-morse>` integration including setup to move camera with vehicles
 - Create new vehicle models for the Morse simulator, including boats, planes and copters
+- Improve :ref:`Gazebo simulator <using-gazebo-simulator-with-sitl>` integration including json protocol, Gazebo9, and new sensors set
 - `MathWorks SimuLink <https://www.mathworks.com/products/simulink.html>`__ interface to ArduPilot SITL
-- `AirSim drone simulator <https://github.com/Microsoft/AirSim/>`__ support for ArduPilot SITL
 - Build system improvements, specifically fixing dependency handling and speeding up the waf build
-- Improvements to the `MAVProxy GCS <https://github.com/ArduPilot/MAVProxy>`__. Adding a parameter editor module, improving waypoint editor. Requires strong python skills.
+- Improvements to the `MAVProxy GCS <https://github.com/ArduPilot/MAVProxy>`__. Better multivehicle support, performance improvement. Requires strong python skills.
```

Comment:
We should check these items for MAVProxy are what Tridge wants the focus to be on

Reply:
Parameter editor was already done last year, I just update with some stuff we speak lately.